standard/tier/wikis/arenafps/tier_data.lua

```diff
@@ -0,0 +1,143 @@
+---
+-- @Liquipedia
+-- wiki=arenafps
+-- page=Module:Tier/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = {
+		{
+			value = '1',
```

Comment:
Do we want this to a be a number instead of string?

Reply:
atm no i think
since atm we expect string returns from lpdb in a bunch of modules
once we have consumers using the tier lib we can chnage it i think